test/expected/agg_bookends_results_optimized.out

```diff
@@ -0,0 +1,199 @@
+-- Copyright (c) 2016-2018  Timescale, Inc. All Rights Reserved.
+--
+-- This file is licensed under the Apache License,
+-- see LICENSE-APACHE at the top level directory.
+SET timescaledb.disable_optimizations= 'off';
+\set ECHO none
```

Comment:
It's impossible to understand/verify this golden file without seeing the queries.

Reply:
Fixed it to print statements as well